test/integration/clientset_test.go

```diff
@@ -568,17 +770,23 @@ func testClusterServiceClassClient(sType server.StorageType, client servicecatal
 	return nil
 }
 
-// TestClusterServicePlanClient exercises the ClusterServicePlan client.
-func TestClusterServicePlanClient(t *testing.T) {
+// TestClusterServiceClassClient exercises the ClusterServiceClass client.
```

Comment:
Is this test for namespaced classes or cluster scoped class?

Reply:
Aligned all types/names/comments now (I hope!).